docs/versions.mdx

```diff
@@ -44,7 +44,13 @@ Versions are named with their creation time by default and have the version crea
 
 ### Initial versions
 
-An initial version is created when a new machine is created or imported. Autosaved machines are is “Initial version” and have a Stately Bot icon.
+An initial version is automatically made for new machines created when:
+
+- you import a machine from code
+- you fork a machine from an existing machine or project
+- you duplicate an existing machine or project
```

Comment:
Might be more accurate to say "you duplicate an existing machine in its project" 

I don't think we "duplicate" projects yet although that could be cool, maybe.